source/config/featureFlagEnums.py

```diff
@@ -67,6 +67,23 @@ def __bool__(self):
 		return self == BoolFlag.ENABLED
 
 
+class ParagraphNavigationFlag(DisplayStringEnum):
+	def _displayStringLabels(self):

```

Comment:
This is missing a property decorator

```suggestion
	@property
	def _displayStringLabels(self):
```